service/mapper/transitions_test.go

```diff
@@ -1057,9 +796,10 @@ func baselineFSM(t *testing.T, status Status) (*Transitions, *State) {
 			IndexPayloads:     true,
 			IndexCollections:  true,
 			IndexTransactions: true,
+			IndexResults:      true,
 			IndexEvents:       true,
 		},
-		log:   testLog,
+		log:   mocks.GenericLogger,
```

Comment:
I wonder if it's worth making the difference between `GenericXXX` and `NoopXXX`? Generic logger implies it does something generic, while it really does nothing.

Reply:
Sure, renaming it to `NoopLogger` makes sense here.